src/components/KSkeleton/KSkeleton.vue

```diff
@@ -115,9 +115,12 @@ const props = defineProps({
   },
 })
 
-const isVisible = ref(false)
+const isVisible = ref(props.delayMilliseconds === 0)
 
 onMounted(() => {
+  if (props.delayMilliseconds === 0) {
```

Comment:
```suggestion
  if (isVisible.value) {
```

Seems more natual this way